README.md

```diff
@@ -22,9 +22,9 @@ Options:
 
 `config.ignition.config_obj`: Set equal to `config.vm.provider :virtualbox`
 
-`config.ignition.config_img`: Set to desired location of generated image (optional)
+`config.ignition.drive_path`: Set to desired path of generated config drive (optional)
```

Comment:
I would call this `drive_root` or similar since it isn't actually the path to the drive.